packages/expo-cli/e2e/__tests__/start-test.ts

```diff
@@ -27,3 +28,34 @@ test('start --offline', async () => {
   });
   cli.stderr.pipe(process.stderr);
 });
+
+test('start --offline with existing packager info', async () => {
+  await ProjectSettings.setPackagerInfoAsync(tempDir, {
+    devToolsPort: 19002,
+    expoServerPort: 19001,
+    packagerPort: 19000,
+    packagerPid: 1337,
+  });
+
+  const promise = spawnAsync(EXPO_CLI, ['start', '--offline'], { cwd: projectDir });
+  const cli = promise.child;
+
+  cli.stderr.pipe(process.stderr);
+  cli.stdout.on('data', data => {
+    process.stdout.write(data);
```

Comment:
On my computer, the `on('data')` event is not triggered if chained with `.pipe(process.stdout)`. This achieves the same result without doing additional stuff.

Reply:
Ville helped me out, I'll change the other test as well using `.pipe` after `.on` 😄